test/e2e_test.go

```diff
@@ -53,11 +53,17 @@ import (
 	logtesting "knative.dev/pkg/logging/testing"
 )
 
+var namespace string
+
+func init() {
+	namespace = os.Getenv("namespace")
```

Comment:
Does this need a default value?

Reply:
Good catch, adding default value as `tekton-chains` if env is empty